example/pages/[pageId].tsx

```diff
@@ -72,8 +73,28 @@ export default function NotionPage({ recordMap }) {
         recordMap={recordMap}
         fullPage={true}
         darkMode={false}
+        vercelImages={true}
         rootDomain='localhost:9090' // used to detect root domain links and open this in the same tab
         components={{
+          image: ({
+            onLoad,
+            src,
+            alt,
+            height,
+            width,
+            objectFit,
+            unoptimized,
+          }) => (
+            <Image
+              src={src}
+              alt={alt}
+              height={height}
+              width={width}
+              objectFit={objectFit}
+              unoptimized={unoptimized}
+              onLoad={onLoad}
+            />
```

Comment:
Notion images has a **"&width="** query for resizing. A similar method can be used for both responsive images and blurred placeholders.

```js
// Loader
const notionImageLoader = ({ src, width }) => {
  return `${src}&width=${width}`
}

// Image component
<Image
  loader={notionImageLoader}
  src={src}
  alt={alt}
  height={height}
  width={width}
  placeholder="blur"
  blurDataURL={`${src}&width=16`}
  ...
/>
```

Reply:
Yeah, this could be really nice to use. We may not need Vercel image optimization then?

Reply:
I think as @normdoow says, it could be a really nice feature but I think we can let the user chose if use a loader or not.